Adding storage gloss terms (#335)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

*See also*:

[[initiator]]
==== image:images/yes.png[yes] initiator (noun)

> *Description*: The consumer of storage from a target. The initiator is typically a server with an adapter card. The initiator “initiates” a connection over the network fabric to one or more ports on your storage system, which are called target ports.

Reviewer: ```suggestion
*Description*: An _initiator_ is the consumer of storage from a target. An nititator is typically a server with an adapter card. The initiator starts a connection over the network fabric to one or more target ports on a storage system.
```
- The phrase `network fabric` doesn't make any sense to me.
